base/exports.jl

```diff
@@ -1063,3 +1063,64 @@ export
     @view,
     @views,
     @static
+
+# TODO: use normal syntax once JuliaSyntax.jl becomes available at this point in bootstrapping
+eval(Expr(:public,
```

Comment:
Not required, but it would perhaps be nice to arrange this list in the same order as the `export` list above - ie, per topic?  At the moment this list looks strangely random and it's hard to get an idea of which Base "API topics" are covered.

Reply:
It's pretty much alphabetical right now, as an artifact of how I generated the list. Yes, topical makes more sense.

Reply:
Grouped